### PR TITLE
enhance etcd-dump-db to display keys in meta more friendly

### PR DESCRIPTION
**Pull request by ahrtr**
Both `consistent_index` and `term` are uint64 values, so it makes more sense to display them as uint64. When the `--decode` flag is provided, then both of them are displayed as uint64 values, otherwise keep the existing behavior. 

See example output below,

```
$ ./etcd-dump-db iterate-bucket ~/tmp/etcd/default.etcd/member/snap/db meta --decode
key="term", value=3
key="consistent_index", value=6
key="confState", value="{\"voters\":[10276657743932975437],\"auto_leave\":false}"
```

### CODE REVIEW

**Comment by ahrtr** [comment]
cc @serathius @spzala @ptabor 
FYI. manually verified that it's working well.

File: tools/etcd-dump-db/backend.go [hunk 1]
```diff
@@ -125,6 +127,14 @@ func authUsersDecoder(k, v []byte) {
 	fmt.Printf("user=%q, roles=%q, option=%v\n", user.Name, user.Roles, user.Options)
 }
 
+func metaDecoder(k, v []byte) {
+	if string(k) == string(schema.MetaConsistentIndexKeyName) || string(k) == string(schema.MetaTermKeyName) {
+		fmt.Printf("key=%q, value=%v\n", k, binary.BigEndian.Uint64(v))
+	} else {
+		fmt.Printf("key=%q, value=%q\n", k, v)
```
Comment (serathius) [inline tools/etcd-dump-db/backend.go:134]: This is a fallback to default decoder from line 160. We should ensure that this code lines match. Can you create a function `defaultDecoder` and use in both places?

**Review by serathius — CHANGES_REQUESTED** [review]
Think the change makes sense, however for clarity could you show examples of output from both before and after the change. This should make the improvement clearer.

File: tools/etcd-dump-db/backend.go (hunk 1 above)
Comment (ahrtr) [reply tools/etcd-dump-db/backend.go]: Done & Thanks.

**Comment by ahrtr** [comment]
> Think the change makes sense, however for clarity could you show examples of output from both before and after the change. Just paste them in a comment on the PR. This should make the improvement clearer.

See examples below.

Before the change (existing behavior):
```
$ ./etcd-dump-db iterate-bucket ~/tmp/etcd/default.etcd/member/snap/db meta
key="term", value="\x00\x00\x00\x00\x00\x00\x00\x03"
key="consistent_index", value="\x00\x00\x00\x00\x00\x00\x00\x06"
key="confState", value="{\"voters\":[10276657743932975437],\"auto_leave\":false}"

$ ./etcd-dump-db iterate-bucket ~/tmp/etcd/default.etcd/member/snap/db meta --decode
key="term", value="\x00\x00\x00\x00\x00\x00\x00\x03"
key="consistent_index", value="\x00\x00\x00\x00\x00\x00\x00\x06"
key="confState", value="{\"voters\":[10276657743932975437],\"auto_leave\":false}"
```

After the change (new behavior):
```
$ ./etcd-dump-db iterate-bucket ~/tmp/etcd/default.etcd/member/snap/db meta
key="term", value="\x00\x00\x00\x00\x00\x00\x00\x03"
key="consistent_index", value="\x00\x00\x00\x00\x00\x00\x00\x06"
key="confState", value="{\"voters\":[10276657743932975437],\"auto_leave\":false}"

$ ./etcd-dump-db iterate-bucket ~/tmp/etcd/default.etcd/member/snap/db meta --decode
key="term", value=3
key="consistent_index", value=6
key="confState", value="{\"voters\":[10276657743932975437],\"auto_leave\":false}"
```

**Review by serathius — APPROVED** [review]

**Comment by serathius** [comment]
cc @ptabor @spzala @ahrtr

**Review by spzala — APPROVED** [review]
Nice. Thanks @ahrtr 
lgtm

**Pull request merged**